browser_utils.js

```diff
@@ -814,6 +814,41 @@ async function workaround_setContent(page, html) {
     await waiter;
 }
 
+/**
+ * Intercept browser requests
+ * @param {import('puppeteer').Page} page
+ * @param {(request: import('puppeteer').Request) => Promise<void> | void} fn
+ */
+async function interceptRequest(page, fn) {
+    if (!page._pentf_intercept_handlers) {
+        try {
+            await page.setRequestInterception(true);
+        } catch(err) {
+            // There is no way to check if request interception
```

Comment:
Can we check the code of the exception here? I'm wary that this would suppress unrelated errors otherwise.

Reply:
Just checked and I mixed up the exception of defining multiple handlers with just setting this option. Calling `page.setRequestInterception()` multiple times works and doesn't throw. My bad 🙈